.travis-bazelrc

```diff
@@ -10,9 +10,9 @@ build --spawn_strategy=standalone --genrule_strategy=standalone
 #build:remote --google_credentials=/tmp/service-account.json
 
 # Set some build options for travis container.
-build --local_resources=1536,1.5,0.5
```

Comment:
any reasons on why we need to increase the computing resources here?

Reply:
nope, I'm trying to debug the build faillures on travis, locally without using `.travis-bazelrc` all goes fine. But even with increased resources it keeps failing https://travis-ci.org/prysmaticlabs/prysm/jobs/408046583. Reverted.